src/gui/widgets/text_box_base.cpp

```diff
@@ -213,7 +204,7 @@ void text_box_base::copy_selection(const bool mouse)
 	}
 
 	unsigned end, start = selection_start_;
-	const std::string txt = text_.text();
+	const std::string txt = get_use_markup() ? plain_text() : text_.text();
```

Comment:
Outside the scope of this PR, but copying to clipboard *really* shouldn't be making a copy of the whole text. We should change `utf8::index` to take a `string_view`, at least, even if we need to make a copy of the selected area for `copy_to_clipboard` (which depends on whether `SDL_SetClipboardText` expects a null-terminated string, I think...)

Reply:
> I wonder if it would be possible to create a string range that ignores pango markup...

Sure. It would be a little tricky though, and I think it would also violate user expectations. If you select a range of text that has bolded words and paste it into a word processor, you kind of expect the bolded words to be preserved. So if we really want to support copying of rich text, I think we should figure out how to actually copy the styles as well.